src/renderer/components/select/select.tsx

```diff
@@ -22,7 +22,8 @@ export interface SelectOption<T = any> {
   label?: React.ReactNode;
 }
 
-export interface SelectProps<T = any> extends ReactSelectProps<T>, CreatableProps<T> {
+export interface SelectProps<T = any> extends ReactSelectProps<OptionTypeBase, boolean, GroupTypeBase<OptionTypeBase>> {
+  options?: readonly any[];
```

Comment:
```suggestion
export interface SelectProps<T = OptionTypeBase> extends ReactSelectProps<T> {
  options?: readonly OptionProps<T, boolean>[];

```

Reply:
And again, `OptionTypeBase` shouldn't be here..

Reply:
Thanks for improvements!

Reply:
We're passing different types into `options` and then converting them inside `this.options()` getter. I suggest in previous iteration an `options` prop was exactly as `any[]`.

Reply:
The good thing we can do is to get rid from `autoConvert` functions, always pass proper `options` and `value` types to this component. I've tried a couple of iterations - they went much out of scope of current PR.